### PR TITLE
support webapp windows container

### PR DESCRIPTION
**Pull request by weidongxu-microsoft**
fix https://github.com/Azure/azure-libraries-for-net/issues/1146

### CODE REVIEW

File: Tests/Fluent.Tests/WebApp/WebAppsTests.cs [hunk 1]
```diff
@@ -156,5 +156,43 @@ public void WebAppWithoutPhp()
...
+                        .WithNewResourceGroup(GroupName1)
+                        .WithPricingTier(PricingTier.PremiumP1v3)
+                        .WithOperatingSystem(OperatingSystem.Windows)
+                        .Create();
+
+                    var webApp = appServiceManager.WebApps.Define(WebAppName1)
+                        .WithExistingWindowsPlan(appServicePlan)
+                        .WithExistingResourceGroup(GroupName1)
+                        .WithPublicDockerHubImage("mcr.microsoft.com/azure-app-service/samples/aspnethelloworld:latest")
+                        .Create();
```
Comment (weidongxu-microsoft) [inline Tests/Fluent.Tests/WebApp/WebAppsTests.cs:185]: Serve as sample as well.

From Portal, web app with windows docker seems had to use PremiumP1v3 or higher.

File: Tests/Fluent.Tests/WebApp/WebAppsTests.cs [hunk 2]
```diff
@@ -156,5 +156,43 @@ public void WebAppWithoutPhp()
                 }
             }
         }
+
+        [Fact]
+        public void CanCRUDWebAppWithContainer()
```
Comment (ChenTanyi) [inline Tests/Fluent.Tests/WebApp/WebAppsTests.cs:161]: may be better `...WithWindowsContainer`?

**Review by ChenTanyi — APPROVED** [review]

**Pull request merged**